src/rpc/blockchain.cpp

```diff
@@ -981,7 +983,7 @@ static UniValue gettxoutsetinfo(const JSONRPCRequest& request)
                         {RPCResult::Type::NUM, "transactions", "The number of transactions with unspent outputs"},
                         {RPCResult::Type::NUM, "txouts", "The number of unspent transaction outputs"},
                         {RPCResult::Type::NUM, "bogosize", "A meaningless metric for UTXO set size"},
-                        {RPCResult::Type::STR_HEX, "hash_serialized_2", "The serialized hash"},
+                        {RPCResult::Type::STR_HEX, "hash_serialized_2", "The serialized hash (only present if 'hash_serialized_2' hash_type is chosen)"},
```

Comment:
Maybe open a separate PR to see if there's support for deprecating `hash_serialized_2` and switch the default to `NONE` (pre-ACK, unless we find someone who actually uses this)

Reply:
Hm, at least BTCPayServer uses it for FastSync: https://github.com/btcpayserver/btcpayserver-docker/tree/master/contrib/FastSync. They say in the docs to compare the whole `gettxoutsetinfo` but the hash is really what ensures there is nothing malicious going on. But I am not sure how much that feature is actually used. I will reach out to them to get an opinion.

Reply:
Is there any benefit of aggressively deprecating this? With the option to select no hash there should be no downside to just leaving the default as is

Reply:
Most people won't realise how slow this command is by default until they already issued it.